py-polars/docs/requirements-docs.txt

```diff
@@ -3,6 +3,10 @@
 numpy
 pandas
 pyarrow
+seaborn
+plotly
+altair
```

Comment:
These can be removed, as they are already in `docs/requirements.txt` (where they should be).

Reply:
Done